deployments/base/kustomization.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+commonLabels:
+  app: boskos
+
+resources:
+  - crd.yaml
+  - deployment.yaml
+  - rbac.yaml
+  - service.yaml
+
+vars:
```

Comment:
Why do we define this here? To not repeat ourselves?

Reply:
I initially did this so you could override the namespace and have the command-line arguments updated appropriately, but looking at the openshift examples, using downward API is even easier, so I've switched to that.